modules/mtbseq/mtbseq_cohort.nf

```diff
@@ -33,11 +29,11 @@ process MTBSEQ_COHORT {
     gatk-register ${gatk_jar}
     sleep 10
     mkdir Joint
-    MTBseq --step TBjoin --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name}
+    MTBseq --step TBjoin --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name} || true
     mkdir Amend
-    MTBseq --step TBamend --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name}
+    MTBseq --step TBamend --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name} || true
     mkdir Groups
-    MTBseq --step TBgroups --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name}
+    MTBseq --step TBgroups --samples ${samples_tsv_ch} --project ${params.mtbseq_project_name} || true
```

Comment:
@Mxrcon , this doesn't look nice since it mixes error handling with the actual code.

Yes, we could use it a possible solution but let's explore a bit more.

Reply:
yes! I agree to, I have 3 other possible solutions and I would like to discuss them:
1. Instruct the bash script to always emit an exit code with number 0
2. Instruct the nextflow part to assign the exit code as 0 always
3. I'm not sure but is there a way of nextflow ignore error exit codes and assign the process as concluded if the outputs are generated? (this would be the perfect solution) 

I've been reading the documentation and not sure about the 3rd one

Reply:
I prefer the third one, for sure. However, as we have seen the error codes are different across executors so perhaps we could simply do 

```
  errorStrategy = { task.exitStatus == 13 ? 'retry' : 'terminate' }

```

Maybe, using this we could retry some combinations like `task.exitStatus in [1,2,3]` etc. Let's see.

Reply:
The problem with forcing the bash part to always exit successfully is that if it really fails - we won't know at all and NF would tell us that it completed successfully.

Reply:
But with my own experience with this process, It would take too much retries and the result is the same: a new retry until reach the max retries. because it'll throw the error code again :( the `maxError` directive won't seem to fix this too. You think that only adding this `errorStrategy` is enough for the moment?

Reply:
>  You think that only adding this errorStrategy is enough for the moment

Davi, the problem is that mtbseq send the bash message that it is failed, but sometimes it actually completes and send the same message.

Reply:
I understand, I'd like to have a way to just instruct to bash ignore mtbseq, set+e should do this and the output beign produced should trigger a completion mark on nf